### ISSUE TITLE
Use gettimeoffday() instead off _ftime()?

### ISSUE THREAD

**Issue by vext01** [FrodeSolheim/fs-uae#233]
Hi,

Whilst porting for OpenBSD, I realised that fs-uae uses `_ftime()`.

Would it be more portable to use `gettimeofday()`?

[Patch](https://github.com/jasperla/openbsd-wip/blob/master/emulators/fs-uae/patches/patch-src_blkdev_cdimage_cpp).

**Comment by FrodeSolheim** [comment]
Yes and no, I think gettimeofday is not available on Windows. Maybe a higher-level wrapper function should be used which can use either ftime or gettimeofday depending on what's available.

(Without looking too much at the code in question, it might also be better to use a monotonic timer instead, if the goal is to track elapsed time).
